app/components/Loading.js

```diff
@@ -12,38 +12,22 @@ const styles = {
   },
 };
 
-export default class Loading extends React.Component {
-  constructor(props) {
-    super(props);
+export default function Loading({ text = "Loading", speed = 300 }) {
+  const [content, setContent] = React.useState(text);
```

Comment:
not a big deal, but you could import `useState` at the top of the page like this:

`import React, {useState} from 'react'`

and then here, you can say:

`const [content, setContent] = useState(text)`